### PR TITLE
Fix minor documentation issues.

### PR DESCRIPTION
**Pull request by snmgian**

### CODE REVIEW

File: lib/ruy/outcome.rb [hunk 1]
```diff
@@ -1,24 +1,39 @@
 module Ruy
+  # Represents a possible result (outcome) of evaluating a rule set
+  #
+  # Additionally, an outcome can be composed of rules. In that case, the outcome will evaluate to
```
Comment (migl) [inline lib/ruy/outcome.rb]: Additional lines in description need to be indented in order for YARD to understand they belong to the same comment block.

File: lib/ruy/outcome.rb [hunk 2]
```diff
@@ -1,24 +1,39 @@
...
+  # its value if all of them are evaluated successfully.
```
Comment (migl) [inline lib/ruy/outcome.rb]: Additional lines in description need to be indented in order for YARD to understand they belong to the same comment block.

File: lib/ruy/outcome.rb [hunk 3]
```diff
@@ -1,24 +1,39 @@
...
 
+    # @param value The value of this outcome
     def initialize(value)
       @value = value
       @rule = Ruy::Rule.new
     end
 
+    # Returns the value of this outcome is all of the conditions succeed.
+    #
+    # It also returns the value when there's no conditions.
```
Comment (migl) [inline lib/ruy/outcome.rb]: Same here.

File: lib/ruy/outcome.rb (hunk 2 above)
Comment (snmgian) [inline lib/ruy/outcome.rb]: I think the indentation rule applies to attributes. 

Here's a description sample from YARD codebase: https://github.com/lsegal/yard/blob/master/lib/yard/parser/source_parser.rb#L255

File: lib/ruy/outcome.rb (hunk 3 above)
Comment (snmgian) [inline lib/ruy/outcome.rb]: https://github.com/lsegal/yard/blob/master/lib/yard/parser/source_parser.rb#L255

**Pull request merged**